frontend/simple/js/state.js

```diff
@@ -92,6 +95,29 @@ const getters = {
 }
 
 const actions = {
+  async login (
+    {dispatch, commit, state}: {dispatch: Function, commit: Function, state: Object},
+    user: string
+  ) {
+    dispatch('loadSettings', user)
+    Vue.events.$once('loaded', async function () {
```

Comment:
This is unnecessary since you can save a lot of lines by awaiting the dispatch. Same goes for below in logout.